client/src/lib/transformers/form.ts

```diff
@@ -0,0 +1,11 @@
+import type { SelectOption } from "src/definitions/form";
+
+export const toSelectOptions = (labelsMap: {
+  [key: string]: any;
+}): Array<SelectOption> => {
+  return Object.keys(labelsMap).map((item, index) => ({
```

Comment:
@Volubyl On dirait que `id` est resté ici bien qu'il ait été retiré de l'interface... Et pourtant TypeScript ne le voit pas ? Est-ce qu'il manque ceci... ?

```suggestion
  return Object.keys(labelsMap).map((item, index): SelectOption => ({
```

Reply:
bien vu! Je pensais que c'était à cause du `any` mais en fait non.

Ta proposition me semble la meilleure option